src/gatsby-theme-docz/wrapper.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Helmet } from "react-helmet-async";
+
+// Note: We are using react-helmet-async here instead of react-helmet
```

Comment:
The comment would help clarity if it included why this is being used.

Reply:
The links below cover the issue. Would it be better to call them out somehow?

Reply:
Sean 👀